docs/index.md

```diff
@@ -30,19 +30,19 @@ Since lakeFS is compatible with the S3 API, all popular applications will work w
 
 <img src="assets/img/s3_branch.png" alt="lakeFS s3 addressing" width="60%" height="60%" />
 
-## Here's why you need it:
+## Use-cases:
 
 ### Development Environment for Data
 * **Experimentation** - try tools, upgrade versions and evaluate code changes in isolation. By creating a branch of the lake you get an isolated snapshot of the lake you can experiment with while others are not exposed. You can compare branches with different experiments or compare your branch to the main branch of the lake.  
 * **Reproducibility** - go back to any point in time to a consistent version of your data lake. By making commit history available for a configurable duration - you can read from the lake at any point in time, compare changes made, and safely roll back if necessary.
 
-[Learn more](https://github.com/treeverse/lakeFS/blob/master/docs/branching/recommendations.md#development-environment){:id="user-content-learn-more-env"}
+[Learn more](https://docs.lakefs.io/usecases/data-devenv.html){:id="user-content-learn-more-env"}
```

Comment:
```suggestion
[Learn more](./usecases/data-devenv.md){:id="user-content-learn-more-env"}
```
So that it works also on dev envs, same for other links